### ISSUE TITLE
Docker pour les développeurs PHP

### ISSUE THREAD

**Issue by erichard** [erichard/blog#2]

**Comment by jjab2020** [comment]
Bonjour, je vous remercie pour cet article, il est très utile.
J'ai une question, si vous voulez bien m'aider bien sur la génération de self-signed certificate pour chaque domaine. quelle la commande a utiliser pour générer un certificat pour chaque domaine .
je ne sais pas peut être a l'aide d'open SSl
Merci bien
Bonne journée

**Comment by jjab2020** [comment]
Bonjour,
J'ai essaye votre solution, ça marche très bien , J'ai ajoute un docker dans le composer de dossier generic , c'est bon , mais ma question , pourquoi au niveau connexion phpmyadmin m'affiche un message d'erreur.

 Unknown MySQL server host 'db' (-2)

Merci Beaucoup.

Bonne journée

**Comment by erichard** [comment]
Bonjour @jjab2020 

Le hostname a utiliser est le nom du container dans la stack générique, `pgsql_95` dans mon exemple.

**Comment by jjab2020** [comment]
oui, merci, mais je parle au niveau phpmyadmin, finalement ça fonctionne voici mon code que je pense sera utile :+1: 
phpmyadmin:
        image: phpmyadmin/phpmyadmin:latest
        networks:
            - web
            - database
        ports:
            - 8009:80 
        environment:
            VIRTUAL_HOST: "domaine"
            VIRTUAL_PORT: "8009"
            PMA_HOST : mysql_57

Il faut définir PMA_HOST dans Environnement.

Autre chose est ce que vous avez réussir a faire fonctionner symfony en virtual host.
pour moi symfony fonctionne uniquement en spécifiant le port  c'est un peu étrange

Merci Beaucoup pour ce magnifique travail.

**Comment by jjab2020** [comment]
Bonsoir encore une fois oui maintenant j'arrive a faire fonctionner symfony , merci beaucoup mon ami.

Si vous voulez je partages mon travail 

Salutations

**Comment by pierreboissinot** [comment]
Merci, retour intéressant. Utilisant Docker également pour calquer mes environnements de prod, j'avais essayé d'aller plus loin en utilisant non plus des environnements portables mais des espaces de travail portables avec la techno [Eclipse Che](https://www.youtube.com/watch?v=05dptvJh49M)(installé en local ou remote) mais sans grand succès et avoir de temps pour cela. Avez-vous un retour sur les workspaces portables ?

**Comment by erichard** [comment]
@pierreboissinot Je n'ai jamais été tenté par ce genre de workspace alors je n'ai pas vraiment de retour d'expérience à faire... Comme je travail sur un portable que j'emmène partout je n'ai pas vraiment le besoin.
